rs/tests/run_systest.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# RUN_SCRIPT_ variables are meant for this script only and shouldn't be forwarded
+# to the test driver
+export -n RUN_SCRIPT_ICOS_IMAGES RUN_SCRIPT_UPLOAD_SYSTEST_DEP RUN_SCRIPT_INFO_FILE_VARS RUN_SCRIPT_TEST_EXECUTABLE RUN_SCRIPT_DRIVER_EXTRA_ARGS
```

Comment:
Could this be formatted as:
```suggestion
export -n \
  RUN_SCRIPT_ICOS_IMAGES \
  RUN_SCRIPT_UPLOAD_SYSTEST_DEP \
  RUN_SCRIPT_INFO_FILE_VARS \
  RUN_SCRIPT_TEST_EXECUTABLE \
  RUN_SCRIPT_DRIVER_EXTRA_ARGS
```